test/gtest/common/test_helpers.h

```diff
@@ -79,7 +79,7 @@
 #define ASSERT_UCS_OK_OR_INPROGRESS(_expr) \
     do { \
         ucs_status_t _status = (_expr); \
-        if ((status) != UCS_OK && (_status) != UCS_INPROGRESS) { \
+        if (((_status) != UCS_OK) && ((_status) != UCS_INPROGRESS)) { \
```

Comment:
there are several unrelated fixes in this PR. can you pls put them in separate PR next time? not just several commits. because they are not related to the main topic of the PR - fix tx ops overflow

Reply:
ok.